### ISSUE TITLE
Dependency `phpunit/phpunit` should move from dev to `require` in `composer.json`

### ISSUE THREAD

**Issue by hms5232** [nunomaduro/collision#261]
Hi, I tested my package today and faced an error:

```
In TestCommand.php line 73:
                                                                              
  Running Collision 7.x artisan test command requires at least PHPUnit 10.x.  
                                                                              

Script ./vendor/bin/testbench package:test handling the testbench event returned with error code 1
Script composer lint; composer testbench handling the test event returned with error code 1
Error: Process completed with exit code 1.
```

You can see more detail here: https://github.com/hms5232/laravel-swagger/actions/runs/4273997592/jobs/7440276939 

The environment is PHP 8.1 and Laravel 9. It was passed before today i want to update the forward compatible with Laravel 10.

After debugging, I notice the [`src/Adapters/Laravel/Commands/TestCommand.php` line 73](https://github.com/nunomaduro/collision/blob/v7.x/src/Adapters/Laravel/Commands/TestCommand.php#L73) throw this error and cause my ci test failed.

In my option, if a dependency is not only used in test but also for package working, the dependency should added in `require` not `require-dev`.

In conclusion, I recommend moving `phpunit/phpunit` from `require-dev` to `require`. Otherwise, each package use the `src/Adapters/Laravel/Commands/TestCommand.php` may face same problem like me.

**Comment by nunomaduro** [comment]
No plans for this - as you may use Collision without the "pest" or "phpunit" things.
